releasenotes/notes/update-kayobe-automation-55fc9c5c380d819c.yaml

```diff
@@ -0,0 +1,19 @@
+---
+features:
+  - |
+    Upgrades kayobe-automation submodule to ``86b2840``.
+    Upgrades kayobe-workflows collection to ``v1.1.0``.
+    Kayobe-automation config-diff now runs in parallel and generates both
```

Comment:
I'd normally say we should only backport fixes to Antelope at this point, but this could really help upgrades. What do you think about adding it to Antelope and merging up?

Reply:
Done. Though moving to draft issue with collection in SMSLab.

Reply:
Issue is related to Caracal and version of Ansible used need to ensure `!unsafe` map is used if applying hooks between steps that use `{{ }}`